datafusion-cli/Cargo.lock

```diff
@@ -2395,12 +2395,12 @@ dependencies = [
 
 [[package]]
 name = "indexmap"
-version = "2.7.0"
+version = "2.5.0"
```

Comment:
I don't know why petgraph forces a downgrade of indexmap

Reply:
I think they have submitted a PR to fix it https://github.com/petgraph/petgraph/pull/714